test/TestBase.py

```diff
@@ -208,16 +208,15 @@ def assertBeatable(self, beatable: bool):
 
     # following tests are automatically run
     @property
-    def skip_default_tests(self) -> bool:
+    def run_default_tests(self) -> bool:
         """Not possible or identical to the base test that's always being run already"""
         constructed = hasattr(self, "game") and hasattr(self, "multiworld")
-        return not constructed or (not self.options
-                                   and self.setUp is WorldTestBase.setUp
-                                   and self.world_setup is WorldTestBase.world_setup)
+        return (constructed and self.options) or not (self.setUp.__code__ is WorldTestBase.setUp.__code__
+                                                      or self.world_setup.__code__ is WorldTestBase.world_setup.__code__)
```

Comment:
I believe this is still wrong :D
we want `"can be run" and "is possibly non-default"` - which would be
```python
constructed and (self.options
                 or self.setUp.__code__ is not WorldTestBase.setUp.__code__
                 or self.world_setup.__code__ is not WorldTestBase.world_setup.__code__)
```
i believe
Also stop doing `not (...)` if it makes the code harder to understand xD